src/dotnet-grpc/Commands/CommandBase.cs

```diff
@@ -442,19 +442,7 @@ private static bool IsStreamContentIdentical(Stream remote, Stream local)
 
     private static byte[] GetHash(Stream stream)
     {
-        SHA256 algorithm;
-        try
-        {
-            algorithm = SHA256.Create();
-        }
-        catch (TargetInvocationException)
-        {
-            // SHA256.Create is documented to throw this exception on FIPS-compliant machines. See
-            // https://msdn.microsoft.com/en-us/library/z08hz7ad Fall back to a FIPS-compliant SHA256 algorithm.
-            algorithm = new SHA256CryptoServiceProvider();
-        }
-
-        using (algorithm)
+        using (var algorithm = SHA256.Create())
```

Comment:
`SHA256CryptoServiceProvider` is obsolete. Also, if you look at the linked doc, the error this code catches can only happen on .NET Framework.